src/cmdstan-guide/installation.Rmd

```diff
@@ -1,6 +1,38 @@
 # CmdStan Installation
 
-To install CmdStan you need:
+## Installation via `conda`
+
+CmdStan can be installed via the package management system [conda](https://docs.conda.io/en/latest/)
+via the [conda-forge channel](https://conda-forge.org/). This
+will install a pre-built version of CmdStan along with the required
+dependencies (i.e. a C++ compiler, a version of Make, and required
+libraries) detailed below under [Source installation].
+
+One can create a new conda environment and install CmdStan with
+the following command.
+
+```
+conda create -n cmdstan -c conda-forge cmdstan
```

Comment:
I think this second "cmdstan" is a free name from the user that you call out later.  Either describe here that the second "cmdstan" can be anything and the subsequent command has to match.

Reply:
it's actually the first cmdstan, which is even more reason to explain